google/cloud/bigtable/internal/table_admin.h

```diff
@@ -155,6 +156,55 @@ class TableAdmin {
 
   //@}
 
+  template <typename ResultType>
+  ResultType PollLongRunningOperation(google::longrunning::Operation& operation,
+                                      char const* error_message,
+                                      grpc::Status& status) {
+    auto polling_policy = polling_policy_->clone();
+    do {
+      if (operation.done()) {
+        if (operation.has_response()) {
+          auto const& any = operation.response();
+          if (not any.Is<ResultType>()) {
+            std::ostringstream os;
+            os << error_message << "(" << metadata_update_policy_.value()
+               << ") - "
+               << "invalid result type in operation=" << operation.name();
+            status = grpc::Status(grpc::StatusCode::UNKNOWN, os.str());
+            return ResultType{};
+          }
+          ResultType result;
+          any.UnpackTo(&result);
+          return result;
+        }
+        if (operation.has_error()) {
```

Comment:
This branch is never tested:

https://codecov.io/gh/GoogleCloudPlatform/google-cloud-cpp/pull/994/diff#D9-180

Consider a test that intentionally returns an error.